lib/daru/io/importers/shared/request_log_analyzer_patch.rb

```diff
@@ -0,0 +1,73 @@
+module RequestLogAnalyzerPatch
+  require 'request_log_analyzer'
```

Comment:
This file now is required always when daru-io is required. It should be made `optional_gem` in importer's `initialize`, like other importers do.

Reply:
Facing a problem here, gem name is `request-log-analyzer` but ` require request-log-analyzer` reads to `LoadError` by the `optional_gem` because `require request_log_analyzer` is the correct way. Should I override the `optional_gem`?